internal/rpmmd/repository.go

```diff
@@ -282,3 +290,27 @@ func (pkg *PackageInfo) FillDependencies(rpmmd RPMMD, repos []RepoConfig) (err e
 	pkg.Dependencies, _, err = rpmmd.Depsolve([]string{pkg.Name}, repos, false)
 	return
 }
+
+// FetchChecksum returns a repository checksum for given repo config
```

Comment:
Hm, why cannot the current helpers be used for the same, the way weldr does? I'm probably missing something.

Reply:
Maybe I'm missing something as well, let me again check those other functions.

Reply:
Well, there is a function `FetchPackageList`, but it will be super hard to read the code if I use `FetchPackageList` to get a checksum.